compose.yml

```diff
@@ -197,21 +198,21 @@ services:
         entrypoint:
             - /updateEFO/updateEFO.sh
 
-    restic:
+    restic_s3:
         image: ghcr.io/wikiteq/restic-backup-docker:stream
-        env_file:
-            - ./.env
-            - ./.env.secret.restic
         profiles:
             - production
         restart: unless-stopped
         links:
             - db
         environment:
             - MW_DB_USER=root
-            - MW_DB_PASS=$MW_DB_INSTALLDB_PASS
+            - MW_DB_PASS=/run/secrets/db_root_password
```

Comment:
Same as above - this won't work like that, the variable is expected to contain a password string, not a path to a file containing a password